### PR TITLE
option to output as json

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
for

```sh
treehugger print -f ham/spam.yaml --json | jq -r .SOME_PASSWORD | some-command
```

File: treehugger/cli/print_out.py [hunk 1]
```diff
@@ -35,6 +39,9 @@ def print_out(args):
     else:
         unencrypted_env_dict = env_dict.decrypt_all_encrypted(plain=True)
 
+    if args.json:
+        print(json.dumps(unencrypted_env_dict))
```
Comment (adamchainz) [inline treehugger/cli/print_out.py]: this continues to output in non JSON as well.

**Comment by graingert** [comment]
Incompetence? Laziness?

**Comment by graingert** [comment]
@adamchainz instead of having single-line crash it I've used it to switch between indented or not indented json.

**Comment by adamchainz** [comment]
Merged in 661edb0470a89f4086525ed05ed003470937d328, thanks.

**Pull request closed without merging**